interfaces/cython/cantera/onedim.pyx

```diff
@@ -1214,6 +1130,47 @@ cdef class Sim1D:
 
         return dgdp - np.dot(L, dfdp)
 
+
+    def eval(self, rdt=-1.0, count = 1):
```

Comment:
In function definitions, there should not be spaces between keyword arguments and the `=` sign